docker-compose-shared.yml

```diff
@@ -0,0 +1,17 @@
+# Docker Compose over ride file for FreeCodeCamp
+#
+# Run with:
+#   docker-compose -f docker-compose.yml -f docker-compose-shared.yml up
+#
+version: "2"
+services:
+    db:
+        networks:
+            - shared
+    freecodecamp:
```

Comment:
Less generic than server, other project will refer to the network as `freecodecamp_shared` which is why I chose a more meaningful name.